hazelcast/src/main/java/com/hazelcast/query/impl/getters/MultiResult.java

```diff
@@ -70,6 +70,8 @@
 
     private List<T> results;
 
+    private boolean nullOrEmptyTarget;
```

Comment:
can we have this kind of comment back? https://github.com/hazelcast/hazelcast/pull/10131/commits/9ded149c1085d829813626046f47206b8b375522#diff-08181c4c9677f20321b81b482ff1d796R75